webapp/views.py

```diff
@@ -78,7 +76,7 @@ def delete(self, request, customer_id):
             user = User.objects.filter(id=customer_id)
 
             if not accounts.exists() and not customer.exists() and not user.exists():
-                raise PermissionDenied("Customer does not exist")
+                raise Http404("Customer does not exist")
```

Comment:
because if the customer_id does not succeed, it still does the delete action and sends a status of 200 successfully, the logic is that to delete you have to look for the customer_id and just delete it